vendor/github.com/containerd/containerd/image.go

```diff
@@ -13,6 +13,7 @@ import (
 	"github.com/opencontainers/image-spec/identity"
 	ocispec "github.com/opencontainers/image-spec/specs-go/v1"
 	"github.com/pkg/errors"
+	"github.com/containerd/containerd/snapshots"
```

Comment:
Why to change the vendored code directly rather than using govendor? @wangforthinker

Reply:
Have done it!